### DISCUSSION TITLE
How to define the time-varying rainfall intensity

### DISCUSSION THREAD

**Discussion by 18052121369** [Anura3D/Anura3D_OpenSource#275] [Q&A]
My name is wang, I want to do an example of slope failure caused by rainfall infiltration. I need to define the rainfall intensity that changes over time, not the uniform rainfall. I did not learn how to define the rainfall intensity that changes over time when I studied the tutorial.

**Comment by Air-shark** [comment]
may I ask if you can simulate time-varing rainfall now?I'd like to consult you.
